__tests__/formats/__snapshots__/scssMaps.test.js.snap

```diff
@@ -11,6 +11,7 @@ $size-font-small: 12rem !default;
 $size-font-large: 18rem !default;
 $color-base-red: #ff0000 !default; // comment
 $color-white: #ffffff !default;
+$asset-icon-book: \\"PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9ImN1cnJlbnRDb2xvciIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGNsYXNzPSJmZWF0aGVyIGZlYXRoZXItYm9vayI+PHBhdGggZD0iTTQgMTkuNUEyLjUgMi41IDAgMCAxIDYuNSAxN0gyMCI+PC9wYXRoPjxwYXRoIGQ9Ik02LjUgMkgyMHYyMEg2LjVBMi41IDIuNSAwIDAgMSA0IDE5LjV2LTE1QTIuNSAyLjUgMCAwIDEgNi41IDJ6Ij48L3BhdGg+PC9zdmc+\\" !default;
```

Comment:
Is the double slash here correct?

Reply:
Pinging this, @7studio is this valid scss?

Reply:
@danielstocks It's a valid SCSS syntax but I don't know why I need to use the double `\` :disappointed: to pass the tests. If I remove them, some tests fail... The first `\` escapes the `"` and the second one escapes the first `\`. This trick/hack seems to be needed by the JS test engine.

Reply:
Ohhhhh I think it has to do with our snapshot tests... it is escaping the quote in JS, and NOT in actual output SCSS. --____--